mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html

```diff
@@ -14,7 +14,7 @@
 {
     promise_test(async () => {
         if (callGetUserMedia) {
-            await setMediaPermission();
+            await setMediaPermission("granted");
             await navigator.mediaDevices.getUserMedia({audio : true, video: true});
```

Comment:
This test might be racy and fail wpt stability mode, as mentioned above. We should hold on to the returned stream and call stop on its tracks. This might be simpler to do if we flatten the test first.

Reply:
This looks like an existing issue, so I'd take a follow-up on this if you prefer.